controllers/tests/controller_server/csi_controller_server_test.py

```diff
@@ -1319,8 +1319,8 @@ def test_publish_volume_map_volume_lun_already_in_use(self, storage_agent):
         self.assertEqual(response.publish_context["PUBLISH_CONTEXT_LUN"], '2')
         self.assertEqual(response.publish_context["PUBLISH_CONTEXT_CONNECTIVITY"], "fc")
 
-        self.mediator.map_volume.side_effect = [array_errors.LunAlreadyInUseError("", "")] \
-            * (self.mediator.max_lun_retries + 1)
+        self.mediator.map_volume.side_effect = [array_errors.LunAlreadyInUseError("", "")] * (
+                    self.mediator.max_lun_retries + 1)

```

Comment:
this keeps moving. please calculate `(self.mediator.max_lun_retries + 1)` in a separate line

Reply:
done #560